Makefile

```diff
@@ -0,0 +1,224 @@
+#### PROJECT SETTINGS ####
```

Comment:
Not sure about this makefile. How is this simpler than the existing one? (50 loc)